css/main.css

```diff
@@ -152,11 +150,9 @@ dt {
 }
 
 .column img {
-    width: 400px;
+padding: 20px;
+  object-fit: contain;
```

Comment:
This made the image take up the entire width of the page
width: 600px; should be added back

Reply:
Actually per new info that I didnt know before, width and height **should** be set on the image itself not on css, sorry when we were talking I didn't realize this was for an img tag

Reply:
So @vricci518 I'd rather pass this for now and fix it in a new ticket.

Reply:
LOL disregard everything i said as im late to the party and @rbaia28 already made the change... @rbaia28 this is why we mark things resolved......

Reply:
:o  my bad